### PR TITLE
Added fat jar support, minor refactoring and support to push to redis

### PR DESCRIPTION
**Pull request by adrake-p44**

### CODE REVIEW

File: src/main/java/com/github/jcustenborder/kafka/connect/redis/RedisConnectorConfig.java [hunk 1]
```diff
@@ -153,7 +165,7 @@ public static ConfigDef config() {
         ).define(
             ConfigKeyBuilder.of(DATABASE_CONFIG, ConfigDef.Type.INT)
                 .documentation(DATABASE_DOC)
-                .defaultValue(1)
+                .defaultValue(0)
```
Comment (sryan-p44) [inline src/main/java/com/github/jcustenborder/kafka/connect/redis/RedisConnectorConfig.java:168]: What was this change for?

File: src/main/java/com/github/jcustenborder/kafka/connect/redis/ByteString.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2014-2020 project44. All rights reserved.
```
Comment (sryan-p44) [inline src/main/java/com/github/jcustenborder/kafka/connect/redis/ByteString.java:2]: Is this correct to have on an open source fork? I don't really know.

File: src/main/java/com/github/jcustenborder/kafka/connect/redis/RedisConnectorConfig.java (hunk 1 above)
Comment (adrake-p44) [reply src/main/java/com/github/jcustenborder/kafka/connect/redis/RedisConnectorConfig.java:168]: Typically redis defaults to database 0 when you connect so I set this default to be more in line with something someone familiar with redis would expect.

File: src/main/java/com/github/jcustenborder/kafka/connect/redis/ByteString.java (hunk 2 above)
Comment (adrake-p44) [reply src/main/java/com/github/jcustenborder/kafka/connect/redis/ByteString.java:2]: Yeah me neither

**Pull request merged**